### PR TITLE
[en] rephrase RFC terms

### PR DESCRIPTION
**Pull request by ebiiim**
The MUST keyword is used in the [IETF I-D](https://tools.ietf.org/html/draft-ietf-quic-transport-22#section-8.1).

### CODE REVIEW

**Comment by bagder** [comment]
Yes, the QUIC drafts use the [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) language but this is supposed to be an explainer book/document written in plain English. In plain English we don't write words in all-caps to emphasize them. Therefor, I don't think we should do this change.

**Comment by ebiiim** [comment]
Since there is a "MUST NOT" on the line after this line, I thought this was just a typo.
If this is what you intended, it's ok to close this PR.

> that says that a server MUST NOT send more than three times the size of the

**Comment by bagder** [comment]
I would rather have us rephrase that to not use RFC "lingo"...

**Comment by ebiiim** [comment]
I tried doing it.

**Comment by bagder** [comment]
Thanks!
